relay-server/src/actors/store.rs

```diff
@@ -762,6 +780,16 @@ impl Handler<StoreEnvelope> for StoreForwarder {
                     start_time,
                     item,
                 )?,
+                ItemType::ReplayPayload => {
+                    debug_assert!(topic == KafkaTopic::ReplayPayloads);
+                    let replay_payload = self.produce_attachment_chunks(
```

Comment:
while the client is already sending payloads in chunks, unfortunately, an _individual chunk_ can still be several megabytes, in the case of a checkout where we need to send a lot of DOM information, e.g., so i think we'll still need to produce chunk messages here. 

We could engineer the frontend to do its own splitting of requests within chunks it needs to send, but i'd rather push this complexity to here as opposed to being on the sdk (plus the extra network requests it would require)